<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InventoryViewForm</class>
 <widget class="QWidget" name="InventoryViewForm">
  <property name="windowTitle">
   <string>Inventory</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_root">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_top">
     <item>
      <widget class="QPushButton" name="btnAddProduct">
       <property name="text">
        <string>Add Product</string>
       </property>
      </widget>
     </item>
     <!-- ✅ دکمه گزارش انقضا -->
     <item>
      <widget class="QPushButton" name="btnExpiryReport">
       <property name="text">
        <string>Expiry Report</string>
       </property>
       <property name="toolTip">
        <string>View products near expiry</string>
       </property>
      </widget>
     </item>
     <!-- ✅ دکمه گزارش موجودی -->
     <item>
      <widget class="QPushButton" name="btnInventoryReport">
       <property name="text">
        <string>Inventory Report</string>
       </property>
       <property name="toolTip">
        <string>View complete inventory summary</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_top">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="txtSearchProduct">
       <property name="placeholderText">
        <string>Search product...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tblProducts">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>